src/index.js

```diff
@@ -101,6 +104,7 @@ window.onload = () => {
 
 // Reload the page whenever the innerHTML of the todo list changes
 todoList.addEventListener('DOMSubtreeModified', () => {
+  alert('DOM Modified');
```

Comment:
- [ ] kindly remove the alert function which you have used in your application because it is not professional, Alert dialogs can interrupt the user's flow and may be considered intrusive. you can alternatively create a modal to handle this effect.